lib/storage.js

```diff
@@ -108,6 +108,35 @@ class Storage {
     return archive;
   }
 
+  renameBoardsWithName(oldName, newName) {
```

Comment:
I don't agree with this approach.

The task data manipulation is not done in the Storage class, this class is supposed to read and write the data, not perform logic like this.
You should move this logic to the TaskBoard class itself and just use the Storage to read and write data back.